connect/runtime/src/test/java/org/apache/kafka/connect/runtime/WorkerConfigTest.java

```diff
@@ -56,6 +57,39 @@
             "set X-Frame-Options:DENY, add  :no-cache, no-store, must-revalidate "
     );
 
+    @Test
+    public void testListenersConfigAllowedValues() {
+        Map<String, String> props = baseProps();
+
+        // no value set for "listeners"
+        WorkerConfig config = new WorkerConfig(WorkerConfig.baseConfigDef(), props);
+        assertEquals(LISTENERS_DEFAULT, config.getList(WorkerConfig.LISTENERS_CONFIG));
+
+        props.put(WorkerConfig.LISTENERS_CONFIG, "http://a.b:9999");
+        config = new WorkerConfig(WorkerConfig.baseConfigDef(), props);
+        assertEquals(config.getList(WorkerConfig.LISTENERS_CONFIG), Arrays.asList("http://a.b:9999"));
```

Comment:
The old JUnit `assertEquals` has the expected value as the first parameter and the actual value as the second, so this is backwards. Check other call points, too.

Reply:
copy paste mistake from other test, I fixed both tests.